src/LinuxTracing/PerfEventRecords.h

```diff
@@ -110,6 +116,9 @@ struct __attribute__((__packed__)) perf_event_callchain_sample_fixed {
   perf_event_sample_id_tid_time_streamid_cpu sample_id;
   uint64_t nr;
   // The rest of the sample is a uint64_t[nr] that we read dynamically.
+  // Following this there is:
+  // perf_event_sample_regs_user_sp_ip_arguments regs;
+  // perf_event_sample_stack_user_8bytes stack;
```

Comment:
Instead of just adding to the previous comment, rewrite it in a single style. "The rest of the sample is a uint64_t[nr]" still sounds like the `uint64_t[nr]` fills the rest of the sample entirely.

Reply:
Thanks. Done.